services/galley/src/Galley/Cassandra/TeamFeatures.hs

```diff
@@ -96,14 +96,24 @@ setFeatureStatusNoConfig ::
   Proxy a ->
   TeamId ->
   TeamFeatureStatus 'WithoutLockStatus a ->
+  Maybe TeamFeatureTTLValue ->
   m (TeamFeatureStatus 'WithoutLockStatus a)
-setFeatureStatusNoConfig _ tid status = do
+setFeatureStatusNoConfig _ tid status ttl = do
   let flag = tfwoStatus status
   retry x5 $ write insert (params LocalQuorum (tid, flag))
   pure status
   where
+    minutes = (* 60)
+    hours = minutes . (* 60)
+    days = hours . (* 24)
```

Comment:
These 3 function read nicely, at the same time it's not so clear what they really do. `days` converts days to seconds, right? What about `fromDays` or `toSeconds`?

Reply:
Yeah, the naming is not great, I just copied it over from https://hackage.haskell.org/package/timespan-0.4.0.0/docs/src/Data.Time.TimeSpan.html#days :D

Reply:
it's local names, so names don't need to be awfully long.